pkg/apis/istio/v1beta1/istio_types.go

```diff
@@ -662,7 +673,7 @@ type HTTPProxyEnvs struct {
 // IstioSpec defines the desired state of Istio
 type IstioSpec struct {
 	// Contains the intended Istio version
-	// +kubebuilder:validation:Pattern=^1.6
+	// +kubebuilder:validation:Pattern=^1.
```

Comment:
Is this change intended?

Reply:
It is, the crd should allow 1.6 and 1.7 as well.